ctrd/container.go

```diff
@@ -486,10 +486,10 @@ func (c *Client) createContainer(ctx context.Context, ref, id, checkpointDir str
 		rootFSPath = container.BaseFS
 	} else { // containers created by pouch must first create snapshot
 		// check snapshot exist or not.
-		if _, err := c.GetSnapshot(ctx, id); err != nil {
+		if _, err := c.GetSnapshot(ctx, container.SnapshotID); err != nil {
```

Comment:
We could use combine the `else` and `if` the reduce the ident, right? 

```
    } else if  _, err := c.GetSnapshot(ctx, container.SnapshotID); err != nil {
        return ......
    } else {
        options = append(options, containerd.WithSnapshot(container.SnapshotID))
    }
```

WDYT? @HusterWan

Reply:
I am afraid we cannot do that, we must first ensure the snapshot existence than we add a `option`

Reply:
Agree, my mistake.